src/test/scala/org/broadinstitute/dsde/workbench/sam/api/ManagedGroupRoutesSpec.scala

```diff
@@ -547,4 +532,67 @@ class ManagedGroupRoutesSpec extends FlatSpec with Matchers with ScalatestRouteT
       responseAs[String] shouldEqual "[]"
     }
   }
+
+  "GET /api/group/{groupName}/admin-notifiers" should "succeed with 200 when the group exists and the requesting user is a group admin" in {
+    val samRoutes = TestSamRoutes(resourceTypes)
+    assertCreateUser(samRoutes)
+    assertCreateGroup(samRoutes)
+    assertGetGroup(samRoutes)
+
+    Get(s"/api/group/$groupId/admin-notifiers") ~> samRoutes.route ~> check {
+      status shouldEqual StatusCodes.OK
+    }
+  }
+
+  it should "fail with 404 when the requesting user is a 'member' but not an 'admin'" in {
+    val samRoutes: TestSamRoutes = TestSamRoutes(resourceTypes)
+    assertCreateGroup(samRoutes)
+
+    val newGuy = makeOtherUser(samRoutes)
+    assertCreateUser(newGuy.routes)
+
+    setGroupMembers(samRoutes, Set(newGuy.email), expectedStatus = StatusCodes.Created)
+
+    Get(s"/api/group/$groupId/admin-notifiers") ~> newGuy.routes.route ~> check {
+      status shouldEqual StatusCodes.NotFound
+    }
+  }
+
+  it should "fail with 404 when the requesting user is not in the group" in {
```

Comment:
I think this test should create the group first to make sure that the 404 is returned because of permissions and not simply because the group doesn't exist.

Reply:
`withUserNotInGroup` asserts the creation of the group